api/api.go

```diff
@@ -604,14 +605,34 @@ func OrbLatestVersion(ctx context.Context, logger *logger.Logger, namespace stri
 	return response.Orb.Versions[0].Version, nil
 }
 
+// OrbLatestVersionOrNew wraps OrbLatestVersion and returns "0.0.0" if no version is found
+func OrbLatestVersionOrNew(ctx context.Context, logger *logger.Logger, namespace string, orb string) (string, error) {
+	v, err := OrbLatestVersion(ctx, logger, namespace, orb)
+	if err != nil {
+		matched, er := regexp.MatchString("has never published a revision", err.Error())
```

Comment:
Since we're expecting to handle this error case in a special way I think there should be some kind of error code to identify the error condition. String matching is brittle, primarily because it's not obvious from the API server code that the exact string text is meaningful. It's easy to change without realizing some consumer depends on the exact text.

I did a quick search for graphQL error codes, and the lucinia docs link to http://facebook.github.io/graphql/June2018/#sec-Errors

Can we setup some kind of error code in the API and handle it here? I guess right now `Errors` is a `[]string` but maybe we could make it a type?